### PR TITLE
Created clear-webs.lua

### PR DESCRIPTION
**Pull request by AtomicChicken**
Following a [request on the forums](http://www.bay12forums.com/smf/index.php?topic=178462.0), here's a simple script for clearing the map of webs, similar to `clear-smoke.lua`. I also made it reset the 'webbed' counter on units, for players looking to free their dwarves from giant cave spiders and the like.

### CODE REVIEW

File: clear-webs.lua [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+they settle on the ground.
+
+Usable in both fortress and adventurer mode.
+
+]====]
+
+local webCount = 0
+for i = #df.global.world.items.other.ANY_WEBS-1, 0, -1 do
+  dfhack.items.remove(df.global.world.items.other.ANY_WEBS[i])
+  count = count + 1
```
Comment (myk002) [inline clear-webs.lua:21]: are all instances of `count` supposed to be `webCount`?

File: clear-webs.lua [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+end
+
+for _, unit in ipairs(df.global.world.units.all) do
+  unit.counters.webbed = 0
```
Comment (myk002) [inline clear-webs.lua:25]: would be nice to detect how many units were freed so the player could tell if it worked

Comment (myk002) [reply clear-webs.lua:25]: I also suspect that some players would just want this functionality, not the removal of all webs across the map. perhaps a flag to isolate the behaviors? I'm not sure what the defaults should be, though

Comment (lethosor) [reply clear-webs.lua:25]: I would expect that just clearing this flag would result in the unit getting stuck again when they attempt to move, but I could be wrong.

File: clear-webs.lua (hunk 1 above)
Comment (AtomicChicken) [reply clear-webs.lua:21]: Clearly a typo inserted while I was wondering whether or not to add a unitCount. No idea how it slipped my notice.

File: clear-webs.lua (hunk 2 above)
Comment (AtomicChicken) [reply clear-webs.lua:25]: Simple enough, will do. 

When a non-WEBIMMUNE unit steps on a thick web, the webbed status is set and the web itself is destroyed. Freeing the unit hence does not necessitate the clearing of any webs, though they will of course remain susceptible to being caught in any surrounding webs later on.

**Review by lethosor — APPROVED** [review]

**Pull request merged**